City-of-Helsinki/drupal-module-helfi-azure-fs
# UHF-943: Initial azure blob storage support, removed unused test

To test this:

1. Run `composer require drupal/helfi_azure_fs:dev-UHF-943 --with-all-dependencies` in your project root
2. Create `public/sites/default/local.settings.php` file
3. Add these to local.settings.php file:
```
$schemes = [
  'azure' => [
    'driver' => 'helfi_azure',
    'config' => [
      'name' => 'helfidrupaltest',
      'key' => 'xxxxx',
      'container' => 'helfi-local',
      'endpointSuffix' => 'core.windows.net',
      'protocol' => 'https',
    ],
    'cache' => TRUE,
    'serve_js' => TRUE,
    'serve_css' => TRUE,
  ],
];
$config['helfi_azure_fs.settings']['use_blob_storage'] = TRUE;

$settings['flysystem'] = $schemes;
```
4. Enable `helfi_azure_fs` module if not enabled already
5. Replace `key` with a correct one (PM me on Slack)
6. Add images and files (to media entities for example) and make sure they are served from blob storage
7. Make sure assets (js/css) are served from blob storage
